vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -197,12 +196,11 @@ def _get_page_divs(self, page: Page) -> _PageDivsType:
             else html.H2(id="dashboard-title", hidden=True)
         )
         settings = html.Div(
-            children=dmc.Switch(
-                id="theme_selector",
-                checked=self.theme == "vizro_light",
+            children=dbc.Switch(
+                id="theme-selector",
```

Comment:
Maybe we should introduce a convention about usage of `snake_case` and `kebab-case`.

Reply:
I agree! I fear we will always have a mix though 😄 

In general, it's best practice to use dashes instead of underscores for all CSS IDs, classnames etc. Hence, why I changed it here. https://stackoverflow.com/questions/7560813/why-are-dashes-preferred-for-css-selectors-html-attributes

However, we use underscores in our Python code and all our model IDs. Hence, most of our callback logic (inputs/outputs) uses underscores. If you take a look at our HTML doc, some of the HTML divs have a classname provided by dash automatically and they also use underscores/dashes combined 😄

Reply:
My personal preference is snake case for one reason only: because I'm used to using it 😄 But, I prefer more to associate all IDs with the same case (no matter which one). However, I don't think we need to do this right away and I consider it a really low priority.

Reply:
I think it's right to prefer the dashes here to match the CSS container rule 👍 It is indeed inconsistent with some callbacks but I don't think it's possible to achieve full consistency here without changing everything to underscores, which just looks wrong in CSS. So basically I think it's good as it is and we have to live with the inconsistency.

Reply:
@huong-li-nguyen @antonymilne 

> Hence, most of our callback logic (inputs/outputs) uses underscores. 

> It is indeed inconsistent with some callbacks

I'm curious: Is it possible to change all our IDs from underscores to dash-like, even though it requires changing the internal components IDs alongside with places where they are used in the actions/callbacks?